src/main/kotlin/org/rust/ide/annotator/fixes/AddStructFieldsFix.kt

```diff
@@ -193,7 +199,7 @@ class AddStructFieldsFix(
                 }
                 factory.createExpression(text)
             }
-            else -> factory.createExpression("()")
-        }
+            else -> null
+        } ?: factory.createExpression("()")
```

Comment:
I think it's a bit unclear when I see `null` as a return value in a non-nullable method. Maybe declare `val default = { factory.createExpression("()") }` in the beggining  and use `else -> default()` instead of `else -> null`, or just revert to `else -> factory.createExpression("()")`?